src/server/handlers/api.go

```diff
@@ -150,6 +150,77 @@ func (h APIHhandler) GetMetrics(w http.ResponseWriter, r *http.Request, argv map
 	return err
 }
 
+// GetExports return a list of metrics if prmetheus export format
+func (h APIHhandler) GetExports(w http.ResponseWriter, r *http.Request, argv map[string]string) error {
+	var res []storage.Item
+	var first bool
+	var err error
+
+	// get data from the form arguments
+	if err = r.ParseForm(); err != nil {
+		return err
+	}
+
+	// we only use gauges
+	if typeStr, ok := r.Form["type"]; ok && len(typeStr) > 0 && typeStr[0] != "gauge" {
+		fmt.Fprintln(w, "[]")
+		return nil
+	}
+
+	// get tenant
+	tenant := parseTenant(r)
+
+	// get a list of gauges
+	if tagsStr, ok := r.Form["tags"]; ok && len(tagsStr) > 0 {
+		tags := storage.ParseTags(tagsStr[0])
+		if !validTags(tags) {
+			return errBadMetricID
+		}
+		res, err = h.Storage.GetItemList(tenant, tags)
+	} else {
```

Comment:
I think you can reduce this `else` condition by defining `tags` to be an empty `map` and if `ok && len(tagStr) > 0` set it to what given in the request.
```Golang
tags = map[string]string{} 
if tagStr, ok := r.Form["tags"]; ok && len(tagStr) > 0 {
  tags = storage.ParseTags(tagStr[0])
  if !validTags(tags) {
    return errBadMetricID
  }
}
res, err = h.Storage.GetItemList(tenant, tags)
```

Reply:
👍 Thanks fixed.